### PR TITLE
fix(scheduler): not all images sent to create_rc will have a dash in them

### PR DESCRIPTION
**Pull request by helgi**

### CODE REVIEW

File: rootfs/scheduler/__init__.py [hunk 1]
```diff
@@ -604,13 +604,15 @@ def _create_rc(self, name, image, command, **kwargs):
...
-                image.replace(':', '-')
-            )
-            TEMPLATE = RCB_TEMPLATE
+        image_name = image[image.index(':')+1:]
+        if '-' in image_name:
+            vcs, sha = image_name.split('-')
+            if vcs == 'git' and len(sha) in [8, 40] and app_type == 'web':
+                imgurl = 'http://{}/git/home/{}/push/slug.tgz'.format(
+                    settings.S3EP,
+                    image.replace(':', '-')
```
Comment (arschles) [inline rootfs/scheduler/__init__.py]: is this `image` variable coming from above? want to make sure this isn't supposed to be `image_name`

Comment (helgi) [inline rootfs/scheduler/__init__.py]: In the original code it wasn't modified. Maybe in the code before. Could have a look at git history, sec

Comment (helgi) [inline rootfs/scheduler/__init__.py]: https://github.com/deis/workflow/commit/3de4d0d811afee038b8c73f0cecccd749f511018 didn't use it either. Maybe that is a bug

Comment (arschles) [inline rootfs/scheduler/__init__.py]: no problem. I just looked up, it's a param to the func

**Pull request merged**